README.md

```diff
@@ -4,10 +4,107 @@
 [![Maintainability](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/maintainability)](https://codeclimate.com/github/codeclimate/codeclimate/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/test_coverage)](https://codeclimate.com/github/codeclimate/codeclimate/test_coverage)
 
-# Running in Docker
-`make docker-build`
+## Table of Contents
 
-`docker-compose up`
+1. [Clone the repo](#clone-the-repo)
+1. [Configure local repo with "git-secrets" protection](#configure-local-repo-with-git-secrets-protection)
+1. [Running in Docker](#running-in-docker)
+1. [Deploying the solution](#deploying-the-solution)
 
-This starts up Postgres, and both API & Worker Spring Boot apps.
+## Clone the repo
```

Comment:
@lhanekam Let's omit the instructions on how to clone the repo. Might be too verbose.

Reply:
"Clone the repo" section removed